### PR TITLE
Release 9.3.0

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: website/versions.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [
+  "v9.3.0",
+  "9.3.0",
```
Comment (npauzenga) [inline website/versions.json:3]: I think we need to remove line 3. Looks like you re-added with the "v" 😄

Comment (jennyhliu) [reply website/versions.json]: Thanks!  I initially used 9.3.0, and then changed it to v9.3.0. It's fixed now.

**Comment by jennyhliu** [comment]
ci build and integration test were successful https://ci.earthdata.nasa.gov/browse/CUM-CBA2544-DIS-2

**Review by npauzenga — APPROVED** [review]
Verified the versioning, the changelog, and the docs. Looks good to me @jennyhliu, nice work!

**Pull request merged**